ring/ring.go

```diff
@@ -101,24 +102,52 @@ func (r *Ring) loop() {
 	})
 }
 
-// Get returns a collector close to the hash in the circle.
-func (r *Ring) Get(key uint32) (IngesterDesc, error) {
+// Get returns up to n ingesters close to the hash in the circle.
```

Comment:
Like: Distributor -> Replicator -> Ingesters?

Hmm, could make sense, but one more type. Let's defer that decision for later, I would propose?